shared/scss/base/base.scss

```diff
@@ -392,3 +407,40 @@ a {
         }
     }
 }
+
+/* Popout box with left arrow */
+.modal-box {
+    .modal-box__body {
+        width: 120px;
+        height: 30px;
+        border-radius: 4px;
+        line-height: 1.5;
+        text-align: center;
+        padding-top: 8px;
+        margin-left: 7px;
+        background-color: $white;
+        border: solid 1px #ddd;
+        color: $slate;
+        text-align: center;
+        z-index: 1;
+        float: right;
+    }
```

Comment:
Could you make the centering of the popup a bit more consistent with the button? It looks a bit jarring when the popup appears.

![screenshot from 2018-02-21 15-54-44](https://user-images.githubusercontent.com/1431577/36490575-70d9d2ee-1720-11e8-99b4-2dbe67df2afb.png)